docs/deployment-guide.md

```diff
@@ -1320,9 +1320,9 @@ If the playbook fails for some reason try re-running it with the `-v` option, if
 
 If the playbook completes successfully but the system is not functioning the best place to start is the log files, these can be found in the `/var/log/alfresco` folder on the target hosts. Please note the nginx log files are owned by root as the nginx process is running as root so it can listen on port 80.
 
-[upload-license]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.4/Alfresco-Content-Services/Administer/Licenses/Upload-new-license
-[global-properties]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.4/Alfresco-Content-Services/Configure/Overview/Using-alfresco-global.properties
-[databases]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.4/Alfresco-Content-Services/Configure/Databases
-[alf-keystores]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.4/Alfresco-Content-Services/Administer/Manage-Security/Authorization/Manage-Alfresco-keystores
-[keystore-generation]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.4/Alfresco-Content-Services/Administer/Manage-Security/Authorization/Manage-Alfresco-keystores/Keystore-generation
-[redundancy]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.4/Alfresco-Content-Services/Administer/High-availability-features/Clustering/Recommendations-for-split-architecture/Scenario-Clustering-for-redundancy
+[upload-license]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/25.1/Alfresco-Content-Services/Administer/Licenses/Upload-new-license
+[global-properties]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/25.1/Alfresco-Content-Services/Configure/Overview/Using-alfresco-global.properties
+[databases]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/25.1/Alfresco-Content-Services/Configure/Databases
+[alf-keystores]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/25.1Alfresco-Content-Services/Administer/Manage-Security/Authorization/Manage-Alfresco-keystores
+[keystore-generation]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/25.1/Alfresco-Content-Services/Administer/Manage-Security/Authorization/Manage-Alfresco-keystores/Keystore-generation
+[redundancy]: https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/25.1/Alfresco-Content-Services/Administer/High-availability-features/Clustering/Recommendations-for-split-architecture/Scenario-Clustering-for-redundancy
```

Comment:
@alxgomz what we do with all of those links which are deep links?

Reply:
Not sure.... @anxumalo any advice? would it be ok to use a search based URL instead if these links cannot be used (e.g. https://support.hyland.com/search/all?query=upload+license&value-filters=platform_custom~%2522Alfresco%2522&content-lang=en-US)?

Reply:
Unfortunately, this type of search query is recommended for our documentation portal as it would pose a risk. For example, if the metadata changed in future, the link would be break.

Reply:
OK so let's keep them for now. Hopefully better support for deep link will come before these link break :)
And if they break before the htmlProofer should save us